### PR TITLE
[conf] Allow nemo-qml-plugin-thumbnailer-qt5. Contributes to JB#14512

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

File: allowed_libraries.conf [hunk 1]
```diff
@@ -28,6 +28,7 @@ libz.so.1
 
 # Nemo Libraries
 libnemonotifications-qt5.so.1
+libnemothumbnailer-qt5.so.1
```
Comment (martyone) [inline allowed_libraries.conf]: Which package provides this?

Comment (jpetrell) [inline allowed_libraries.conf]: Package nemo-qml-plugin-thumbnailer-qt5

Comment (martyone) [inline allowed_libraries.conf]: Now I see it is a new addition not yet on the image I checked with

**Comment by jpetrell** [comment]
Can we merge this?

**Pull request merged**